packages/esm-patient-vitals-app/src/common/data.resource.ts

```diff
@@ -47,28 +47,26 @@ export interface ConceptMetadata {
 }
 
 interface VitalsConceptMetadataResponse {
-  results: Array<{
-    setMembers: Array<ConceptMetadata>;
-  }>;
+  setMembers: Array<ConceptMetadata>;
 }
 
 function getInterpretationKey(header: string) {
   // Reason for `Render` string is to match the column header in the table
   return `${header}RenderInterpretation`;
 }
 
-export function useVitalsConceptMetadata() {
+export function useVitalsConceptMetadata(vitalsSignsConceptUUID) {
   const customRepresentation =
```

Comment:
```suggestion
  const { concepts } = useConfig<ConfigObject>();
  const vitalSignsConceptSetUuid = concepts.vitalSignsConceptSetUuid;
  
  const customRepresentation =
```

We could get the config property directly here from the `useConfig` hook and avoid passing it down as an argument in all invocations of `useVitalsConceptMetadata`. This approach:

- Directly destructures the specific UUID you need from the config
- Reduces unnecessary variable declarations
- Makes the code more concise and easier to follow
- Eliminates the need to pass the concept UUID as an argument through multiple function calls

Reply:
@denniskigen getting the UUID directly in one place does seem better than getting it and passing it in everywhere, thanks. I'll change the pull request title to refactor as well.